CHANGELOG.md

```diff
@@ -3,17 +3,36 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
-- Add `plotly[express]` extra for easily installing Plotly Express dependencies [#4644](https://github.com/plotly/plotly.py/pull/4644)
+- Add `plotly[express]` extra for easily installing Plotly Express dependencies [[#4644](https://github.com/plotly/plotly.py/pull/4644)]
+- Add `subtitle` attribute to all Plotly Express traces [[#4830](https://github.com/plotly/plotly.py/pull/4830)].
 
 ### Removed
-- Drop deprecated `pointcloud` and `heatmapgl` traces from the API
-- Drop `tenacity` dependency [#4831](https://github.com/plotly/plotly.js/pull/4831)
-
-### Updated
-
-- Updated plotly.py to use base64 encoding of arrays in plotly JSON to improve performance.
-- Add `subtitle` attribute to all Plotly Express traces
-- Make plotly-express dataframe agnostic via Narwhals [#4790](https://github.com/plotly/plotly.py/pull/4790)
+- Drop deprecated `pointcloud` and `heatmapgl` traces from the API [[#4815](https://github.com/plotly/plotly.py/pull/4815)]
+- Drop `tenacity` dependency [[#4831](https://github.com/plotly/plotly.py/pull/4831)]
+- Drop support for Jupyter Notebook version 6 and earlier [[#4822](https://github.com/plotly/plotly.py/pull/4822)]
+
+### Updated
+- Update plotly.py to use base64 encoding of arrays in plotly JSON to improve performance [[#4470](https://github.com/plotly/plotly.py/pull/4470)].
+- Make plotly-express dataframe agnostic via Narwhals [[#4790](https://github.com/plotly/plotly.py/pull/4790)].
+- Update `go.FigureWidget` to use `anywidget` [[#4823](https://github.com/plotly/plotly.py/pull/4823)]
+- Use modern [native ES6 import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) to load plotly.js bundle instead of requirejs which is [no longer under active development](https://github.com/requirejs/r.js/compare/2.3.6...2.3.7) [[#4736](https://github.com/plotly/plotly.py/pull/4763)]
+- Update Plotly.js from version 2.34.2 to version 3.0.0-rc0 See the [plotly.js CHANGELOG](https://github.com/plotly/plotly.js/blob/master/CHANGELOG.md#300-rc0----2024-11-11) for more information. These changes are reflected in the auto-generated `plotly.graph_objects` module. Notable changes include:
+  - Drop support for deprecated attributes `titlefont`, `titleposition`, `titleside`, and `titleoffset` [[#7212](https://github.com/plotly/plotly.js/pull/7212)].
```

Comment:
Could one still use a sting in place of titles? If no then `title` should also be mentioned for this item.

Reply:
Thank you! Resolved.